common-app-lib/src/cmr/common_app/api/launchpad_token_validation.clj

```diff
@@ -53,7 +29,7 @@
   [request-context]
```

Comment:
on line 28 of this file the validate-launchpad-token description still states 
Ingest will only be allowed if the user is in the NAMS CMR Ingest group
   (done in legacy services) and
Is this statement still true?  If not, could you please change it so that it documents what is true?